docs/content/templates.md

```diff
@@ -69,3 +94,32 @@ I recommend against using a CSS inliner with Cerberus. Here’s why:
 - Inlining leads to code bloat. Not only does this impact download speeds, but some email clients like Gmail and iOS Outlook truncate messages after they exceed a certain file size.
 
 I’m a fan of using snippets, available in most code editors. I understand CSS inliners help many folks, but if you use one with Cerberus, please do so at your own risk. I recommend [Lee Munroe’s CSS inliner](https://htmlemail.io/inline/) and hear good things about [Roadie](https://github.com/Mange/roadie).
+
+<script>
+  (function () {
+    const forms = document.querySelectorAll('.parcel-button');
+    for (let i = 0; i < forms.length; i++) {
+      const form = forms[i];
+      form.addEventListener('click', function (event) {
+        const url = form.getAttribute('data-url');
+        // if we have a url to fetch html from
+        if (!form.elements.data.value && url) {
```

Comment:
Totally silly of me – in the example I forgot to add the else condition:

Add this and then this looks great!
```
else {
form.submit()
}
```

Reply:
Without this, the second time you click on the button it doesn't work

Reply:
@avigoldman [Like so?](https://github.com/TedGoas/Cerberus/pull/282/commits/1dab19f7e4957da393df95329372c2b429994b83)

Reply:
Perfect!